### ISSUE TITLE
Feature request: collapsible admonitions from Obsidian syntax

### ISSUE THREAD

**Issue by SebastianErfort** [ndy2/mkdocs-obsidian-support-plugin#4]
Hi,

It would be great to translate the Obsidian syntax for collapsible call-outs to MkDocs admonitions

```markdown
> [!note]- <title>
> <lines>
```
to
```markdown
??? note "<title>"

    <lines>
```

and 
```markdown
> [!note]+ <title>
> <lines>
```
to
```markdown
???+ note "<title>"

    <lines>
```

I saw something similar was recently added in #2 for a syntax for admonitions I haven't come across yet. (By the way, where is this backtick syntax being used?)

I will try to implement this myself, but I first need to figure out how to have MkDocs use my fork of your plugin and should be working on other things. 😅

Thanks a lot for the plugin, it has been very helpful in integrating my Obsidian notes in a MkDocs documentation system.

Cheers

**Comment by ndy2** [comment]
Thank you for your suggestion!

I implemented it on 0.7.2 and deployed it to pypi.
and [here](https://ndy2.github.io/mkdocs-obsidian-support-plugin/features/admonition-collapsable/) is sample usage of it! 

There are some miss matching in raw text, converted text in the documentation.
But i will fix it soon or later

**Comment by SebastianErfort** [comment]
Nice, thank you for the quick implementation!
